monitoreo/apps/dashboard/management/import_utils.py

```diff
@@ -54,16 +62,15 @@ def import_indicators_tempfile(indicators_file, model, user):
 
 
 def import_indicators(indicators_file, model):
-    types_mapping = {ind_type.nombre: ind_type for
-                     ind_type in IndicatorType.objects.all()}
+    types_mapping = {ind_type.nombre: ind_type for ind_type in IndicatorType.objects.all()}
     with open(indicators_file) as indicators_csv:
         indicators = []
         csv_reader = csv.DictReader(indicators_csv)
         with suppress_autotime(model, ['fecha']):
             with transaction.atomic():
                 for row in csv_reader:
-                    row['indicador_tipo'] = \
-                        types_mapping[row.pop('indicador_tipo')]
-                    indicators.append(model(**row))
+                    model_attributes = {FIELD_TRANSLATIONS[key]: row[key] for key in row}
```

Comment:
utilizar `row.items()` en el comprehension aumentaría legibilidad. me costó un poquito parsear esto

algo como `{FIELD_TRANSLATIONS[key]: value for key, value in row.items()}`